lib/sufia/engine.rb

```diff
@@ -53,6 +53,10 @@ class Engine < ::Rails::Engine
       end
 
       CurationConcerns::CurationConcern.actor_factory = Sufia::ActorFactory
+      # Don't try to load this class until the application has been generated
+      if defined? ::SearchBuilder
+        CurationConcerns::AdminSetService.default_search_builder = Sufia::AdminSetSearchBuilder
+      end
```

Comment:
It breaks when you try to load sufia, before you run `rails g sufia:install`, this just protects against that breakage.